packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -592,25 +598,51 @@ export function MainStackScreen() {
       pincodeType,
       account,
       hasSeenVerificationNux,
+      shouldSkipOnboardingEducationScreen: savedShouldSkipEducationScreen,
     } = mapStateToProps(store.getState())
 
+    // Remove Onboarding Education Screen Experiment: Because remote configs are fetched after the initial route is launched,
+    // The randomization is hardcoded here to achieve a 50/50 split, the value is written into the redux store so the same experience
+    // would persist. This block of code should be removed when the experiment is done.
+    const _shouldSkipOnboardingEducationScreen =
+      savedShouldSkipEducationScreen ?? Math.random() < 0.5
```

Comment:
I think that a more reliable and cleaner way to ensure persistance would be to use a random number generator that uses a seed instead of Math.random() and use deviceId as a seed.

For example, using [seedrandom](https://www.npmjs.com/package/seedrandom)
```
const rng = seedrandom(DeviceInfo.getDeviceId())
const randomNumber = rng() // Always the same random number for a given deviceId
```

Reply:
great suggestion